microsoft-edge/progressive-web-apps-chromium/how-to/share.md

```diff
@@ -71,31 +71,37 @@ async function shareSomeContent(title, text, url) {
 }
 ```
 
-On Windows, the above code will trigger the share dialog, allowing the user to pick an app to receive the shared content.
+On Windows, the above code will trigger the share dialog, allowing the user to pick an app to receive the shared content.  The share dialog is shown below:
+
+:::image type="content" source="../media/windows-share-dialog.png" alt-text="The share dialog on Windows.":::
 
-:::image type="content" source="../media/windows-share-dialog.png" alt-text="The share dialog on Windows." lightbox="../media/windows-share-dialog.png":::
+Once the user has chosen an app to receive the shared content, it is up to this app to handle it any way it chooses.  For example, an email app might use the `title` as the email subject, and use the `text` as the email body.
 
-Once the user has chosen an app to receive the shared content, it is up to this app to handle it any way it chooses. An email app might use the `title` as the email subject and the `text` as the body, for example.
 
-### Sharing files
+<!-- ====================================================================== -->
+## Sharing files
```

Comment:
I don't think changing the h3 in h2 is a good idea in this article.

There are 2 main directions to sharing: sharing from an app, and accepting shared data from another app.
I believe we want to keep these 2 as parent sections to the rest.

For example, this section `Sharing files` "belongs" to the `Sharing content` h2 section. It only makes sense in this direction (sharing from an app).

For the other way around, the `Register as a target`, `Handle GET shared data`, `Handle POST shared data`, and `Handle shared files` sections belong to the `Receiving shared content` h2 section. They don't make sense on their own.

I see you've added the phrase `The following sections cover receiving shared content.` further down to help with this.
So I imagine you made this change so that the table of content displayed on the page would be more useful. Is that correct?

Reply:
As discussed offline, this was done as a way to promote h3s to h2s as a way for them to show up in the TOC.
I feel like it breaks the structure of the document a little bit, but at the same time, is this even really visible to readers? The font-size of h2 and h3 isn't very different.
So I'm good either way.

Reply:
h2s were changed back to h3s, in latest .md file diff (not visible in Outdated diff).